### ISSUE TITLE
Possible bug in openebs/maya's ci/helm_install_openebs.sh file

### ISSUE THREAD

**Issue by AmitKumarDas** [openebs/openebs#1305]
Below is a snippet used in https://github.com/openebs/maya/blob/master/ci/helm_install_openebs.sh

```bash
helm install openebs-charts/openebs --name ci --set apiserver.tag="ci",jiva.replicas="1"
```
However openebs charts' values.yaml has been modified. This modification requires a change in above setting. In other words `apiserver.tag` should be replaced with `apiserver.imageTag`

Refer to current chart's values.yaml
```yaml
# Default values for OpenEBS

## If true, create & use RBAC resources
##
rbacEnable: true
rbac:
  namespace: "openebs"

operator:
  name: "openebs-maya-operator"

image:
  pullPolicy: IfNotPresent

apiserver:
  image: "openebs/m-apiserver"
  imageTag: "0.5.2"
  replicas: 2
  ports:
    externalPort: 5656
    internalPort: 5656
  nodeSelector: {}
  tolerations: {}
  antiAffinity:
    enabled: true
    type: Hard

provisioner:
  image: "openebs/openebs-k8s-provisioner"
  imageTag: "0.5.3-RC1"
  replicas: 2
  mayaportalurl: "https://mayaonline.io/"
  monitorurl: "http://127.0.0.1:32515/dashboard/db/openebs-volume-stats?orgId=1"
  monitorvolkey: "&var-OpenEBS"
  nodeSelector: {}
  tolerations: {}
  antiAffinity:
    enabled: true
    type: Hard

grafana:
  image: "grafana/grafana"
  imageTag: "4.6.3"
  ports:
    externalPort: 3000
    internalPort: 3000
    nodePort: 32515

prometheus:
  image: "prom/prometheus"
  imageTag: "v2.1.0"
  ports:
    externalPort: 80
    internalPort: 9090
    nodePort: 32514

jiva:
  image: "openebs/jiva"
  imageTag: "0.5.2"
  replicas: 3

policies:
  monitoring:
    enabled: true
    image: "openebs/m-exporter"
    imageTag: "0.5.2"
```

**Comment by kmova** [comment]
This has been fixed in 0.5 branch. The same fix needs to get into master. Changing the milestone to 0.6.

**Comment by kmova** [comment]
Fixed in master by:
https://github.com/openebs/maya/pull/299
https://github.com/openebs/maya/pull/311
